Dockerfile

```
@@ -2,27 +2,30 @@
 # BUILDER #
 ###########
 FROM clinicalgenomics/python3.12-venv:1.0 AS python-builder
```

```suggestion
FROM python:3.12-slim-bookworm AS python-builder
```

Ok, but since we are not using virtualenv I think we could just switch to python:3.12-slim-bookworm, it's one less library to maintain in the future

---

Sure; I gave it a spin earlier with bookworm as builder and slim as runner. That way we don't have to install as much I think, but I'll need to clean some docker caches before I can say.. 😁

---

I switched to a bookworm and a slim-bookworm with uv on them, so from astral. With bookworm, we can fetch a few less packages less at least, though I guess the updating of apt and its catalog is the real time hog now. We could consider a base image with the pdf libs preinstalled for scout and chanjo-report. This should be good for now I think.